src/scala/org/pantsbuild/zinc/compiler/Main.scala

```diff
@@ -180,7 +180,13 @@ object Main {
 
     Settings.SettingsParser.parse(preprocessArgs(context.getArgs), Settings()) match {
       case Some(settings) =>
-        mainImpl(settings.withAbsolutePaths(new File(context.getWorkingDirectory)), startTime, n => context.exit(n))
+        // We don't want to reload scala compiler everytime we call nailmain so just the version
```

Comment:
Rather than having the nailMain implementation effectively ignore this setting, IMO, we should remove the setting to help to ensure that the behaviour is consistent between main and nailMain.

Reply:
+1 to that, I don't think this will cause issues with the current usage, so we can get rid of the configurability.

Reply:
Are you referring to the -scala-path and -compiler-bridge-jar settings?

Reply:
I think so? Anything that this change makes unnecessary, because we're loading it from the tool's classpath instead.

Reply:
Okay, I see what you mean. That is do-able I'm pretty sure. The default scala path will be the zinc classpath.